### PR TITLE
Updated the description of Icecoon's Weapons Pack

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**

### CODE REVIEW

File: data/mods/EW_Pack/modinfo.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
     "type": "MOD_INFO",
     "mod_type": "SUPPLEMENTAL",
     "ident": "ew_pack",
-    "name": "Icecoon's Weapons Pack",
-    "description": "For those gun nuts.",
-    "author": "Icecoon",
+    "name": "Icecoon's Arsenal",
+    "description": "For those gun nuts.  Don't have enough near-future firearms in your life?  Add this mod today!",
+    "author": "Icecoon and community",
```
Comment (mugling) [inline data/mods/EW_Pack/modinfo.json]: I think you can actually do an array here

Comment (BorkBorkGoesTheCode) [inline data/mods/EW_Pack/modinfo.json]: How can I array?

Comment (illi-kun) [inline data/mods/EW_Pack/modinfo.json]: Try this:
```"authors": ["Icecoon", "BorkBorkGoesTheCode" ],```
Note that keyword "author" is changed to its plural form.

**Comment by BorkBorkGoesTheCode** [comment]
I've used the array

**Pull request merged**